x-pack/plugins/observability/public/pages/alerts/alerts_flyout/index.tsx

```diff
@@ -31,16 +31,16 @@ import {
 } from '@kbn/rule-data-utils/target/technical_field_names';
 import moment from 'moment-timezone';
 import React, { useMemo } from 'react';
-import type { TopAlert, TopAlertResponse } from '../';
+import type { TopAlert } from '../';
 import { useKibana, useUiSetting } from '../../../../../../../src/plugins/kibana_react/public';
 import { asDuration } from '../../../../common/utils/formatters';
 import type { ObservabilityRuleTypeRegistry } from '../../../rules/create_observability_rule_type_registry';
-import { decorateResponse } from '../decorate_response';
+import { parseAlert } from '../parse_alert';
 import { SeverityBadge } from '../severity_badge';
 
 type AlertsFlyoutProps = {
   alert?: TopAlert;
-  alerts?: TopAlertResponse[];
+  alerts?: Array<Record<string, unknown>>;
```

Comment:
`TopAlertResponse` was only every indirectly defined via the return type of the server-side route registration. The removal of that route meant the type went away as well. The data fetching now happens via the `timeline` search strategy, which doesn't provide much type safety around the data, which is reflected by this `Record<string, unknown>`. The object is decoded using `parseAlert` anyway (which is the successor to `decorateResponse`).

Reply:
Even if we don't have much type safety now, would it make sense to extract `Record<string, unknown>` to a separate type? That way if/when we get better typings it's a simpler change